app/views/home/index.html.erb

```diff
@@ -1,6 +1,13 @@
-<h1>Bail Reform at Berkeley</h1>
-<div class="DonateAndApplyContainer">
-  <button type="button">Apply for Bail</button>
-  <%= button_to "Donate", new_donation_path, method: :get %>
-</div>
-<%= image_tag("HomePageBailReformLogo.png",  :size => "300x200", :alt => "homepage logo") %>
+<h1>Bail Reform at Berkeley</h1>
+<div class="navigationBar">
+  <%= link_to "About Us", about_path, method: :get %>
+  <%= link_to "Meet the Team", team_path, method: :get %>
+  <%= link_to "Get Involved", volunteer_path, method: :get %>
+  <%= link_to "FAQ", faq_path, method: :get %>
+  <%= link_to "Blog", blog_path, method: :get %>
+</div>
+<div class="DonateAndApplyContainer">
+  <%= button_to "Apply for Bail", new_apply_path, method: :get %>

```

Comment:
I actually think these two buttons should be in the navigation bar.

Reply:
Someone else put the buttons there, I just put the nav bar on top. I think it's ok to keep it as is for now and discuss design later (it's not hard to change).

Reply:
They were just buttons because the navbar hadn't been created.  I made them buttons with the intention of moving them when we got to that point.

Reply:
Made a landing/info page for donating/apply and moved the buttons there. The landing pages are what's in the navbar.

Reply:
I'm confused, which page is the landing/info page?

Reply:
should be there now, e.g. ```donation_path``` goes to ```donation#index```. The old button is on that page